app/dashboard/views/support.py

```diff
@@ -0,0 +1,93 @@
+import json
+import urllib.parse
+from typing import Union
+
+import requests
+from flask import render_template, request, flash, url_for, redirect
+from flask_login import login_required, current_user
+from werkzeug.datastructures import FileStorage
+
+from app.dashboard.base import dashboard_bp
+from app.log import LOG
+from app.models import Mailbox
+from app.config import ZENDESK_HOST
+
+VALID_MIME_TYPES = ['text/plain', 'message/rfc822']
+
+
+@dashboard_bp.route("/support", methods=["GET"])
+@login_required
+def show_support_dialog():
+    mailbox = Mailbox.get(current_user.default_mailbox_id)
```

Comment:
We usually use the user email address (`current_user.email`) to get in touch with them so I think we can replace `mailbox.email` here by `current_user.email`.  And as `current_user` is directly available in the html template, we don't then need to pass a variable to the template.

Reply:
Ok. I'll replace `mailbox.email` with `current_user.email` But I'd rather keep the variable in the template. It's used to re-fill the template if the submission is missing some value in the POST route.